cityofaustin/atd-product
# Upgrade to React v17, NextJS 12, and more

Resolves https://github.com/cityofaustin/atd-data-tech/issues/9581

Major upgrades!
- Node v16 (with `nvm` configuration)
- React v17
- NextJS v12

### Steps to test
- use local or [deploy preview](https://deploy-preview-40--atd-product.netlify.app/)
- Click around the projects and product listings, details. Make sure everything renders happily. 
- Check out a project's "Activity" tab and inspect comments
- if local, remember to start with `$ nvm use`.  You may need to delete `/node_modules` entirely.
